src/gource_settings.cpp

```diff
@@ -71,6 +71,7 @@ void GourceSettings::help(bool extended_help) {
     printf("                                   for a number of seconds (default: 3)\n");
     printf("      --disable-auto-skip          Disable auto skip\n");
     printf("  -s, --seconds-per-day SECONDS    Speed in seconds per day (default: 10)\n");
+    printf("      --transition TIMESTAMP       Timestamp for transitions\n");
```

Comment:
Perhaps `Timestamp for seconds per day transition`

Reply:
Will also need a bit of README / man page documentation for how to use this feature.